### PR TITLE
flyway added

### PR DESCRIPTION
**Pull request by jnejkovic**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -5,8 +5,8 @@ services:
     image: mysql:8
     ports:
       - 3306:3306
-    volumes:
-      - ~/apps/mysql:/var/lib/mysql
+#    volumes:
```
Comment (lela777) [inline docker-compose.yml]: Don't leave commented code. Please remove these lines or uncomment them.

File: pom.xml [hunk 2]
```diff
@@ -61,6 +61,21 @@
...
+            <artifactId>spring-boot-starter-actuator</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.flywaydb</groupId>
+            <artifactId>flyway-core</artifactId>
+            <version>8.5.13</version>
+        </dependency>
+        <!-- https://mvnrepository.com/artifact/org.flywaydb/flyway-mysql -->
+        <dependency>
+            <groupId>org.flywaydb</groupId>
```
Comment (lela777) [inline pom.xml:74]: Is this dependency necessary?

File: src/main/resources/db/migration/V1__initial_schema_migration.sql [hunk 3]
```diff
@@ -0,0 +1,53 @@
+    drop table if exists airport;
+
+    drop table if exists flight;
+
+    drop table if exists gate;
+
+    drop table if exists hibernate_sequence;
```
Comment (lela777) [inline src/main/resources/db/migration/V1__initial_schema_migration.sql]: Drop commands are not necessary.

File: src/main/resources/db/migration/V1__initial_schema_migration.sql [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+    create table airport (
```
Comment (lela777) [inline src/main/resources/db/migration/V1__initial_schema_migration.sql]: Please remove empty spaces at the beginning of each command.

**Review by lela777 — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (jnejkovic) [reply pom.xml:74]: Yes, without that dependency I get exception  org.flywaydb.core.api.FlywayException: Unsupported Database: MySQL 8.0

File: src/main/resources/db/migration/U1__initial_schema_rollback.sql [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+DROP TABLE airport;
+
+DROP TABLE flight;
+
+DROP TABLE gate;
+
+DROP TABLE hibernate_sequence;
+
+DELETE FROM airport_gate.flyway_schema_history
```
Comment (lela777) [inline src/main/resources/db/migration/U1__initial_schema_rollback.sql:15]: please remove airport_gate.

**Review by lela777 — APPROVED** [review]

**Pull request merged**